### DISCUSSION TITLE
照片欣赏：工业化社会场景1920 vs 2020

### DISCUSSION THREAD

**Discussion by yibingbingwu** [RoadSideUniversity/ClubHouse2024#32] [General]
刚才那篇文章 #31 中有张照片很有意思，是房贷推销员在拉客户：
![new_factory1](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/30208306/232deaff-54c7-4036-a7b3-bfe4a776b07d)

想到100年前福特公司的场景：
![old_factory](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/30208306/d932c598-9f90-4a39-9389-425b12705a01)

无他，只是感叹而已：有好多不一样的，但还是有好多一样的

**Comment by Lewisrxliu** [comment]
精细社会分工是社会进步的标志之一
